src/material/chips/chip.scss

```diff
@@ -184,7 +184,7 @@
 }
 
 // The ripple container should match the bounds of the entire chip.
-.mat-mdc-chip-ripple {
+.mat-ripple.mat-mdc-chip-ripple {
```

Comment:
Which styles are overwritten by the ripple? Maybe we can move only those under a more specific selector?

Reply:
The regular mat-ripple styles. It depends on the order that teams load their styles. IIRC, some teams have their material-theme.css stylesheet in the `body`

Reply:
But which property from the regular ripple styles overrides the chip styles? I feel like only the `border-radius` might be problematic.

Reply:
https://github.com/angular/components/blob/main/src/material/core/ripple/_ripple.scss#L6

Reply:
<img width="465" alt="Screenshot 2023-07-14 at 12 11 30 PM" src="https://github.com/angular/components/assets/25158423/4f7efb2f-efc2-4dc7-a31d-9ecafae6072c">